### PR TITLE
Remove opentelemetry-proto in .gitsubmodule

### PR DESCRIPTION
**Pull request by YHM404**
opentelemetry-proto was removed in #1713

### CODE REVIEW

**Comment by pellared** [comment]
1. Can you sign EasyCLA?
2. I think you referenced a wrong PR.

**Comment by pellared** [comment]
@YHM404 bump

**Comment by YHM404** [comment]
> 1. Can you sign EasyCLA?
> 2. I think you referenced a wrong PR.

Hi, I already signed EasyCLA. and I recorrect the referenced PR.

**Review by MrAlias — APPROVED** [review]

**Review by hanyuancheung — APPROVED** [review]

**Review by dmathieu — APPROVED** [review]

**Review by pellared — APPROVED** [review]

File: .gitmodules
Comment (pellared) [inline .gitmodules:1]: Why not simply removing the file?

**Pull request merged**